src/bin/am/commands/start.rs

```diff
@@ -548,6 +568,14 @@ async fn start_prometheus(
 
     serde_yaml::to_writer(&config_file, &prometheus_config)?;
 
+    if !no_rules {
```

Comment:
Given the double negative, it would would be better to name this `rules_enabled` or `enable_rules`.